tasks/ceph_manager.py

```diff
@@ -727,11 +731,11 @@ def do_dump_ops(self):
             for osd in self.live_osds:
                 # Ignore errors because live_osds is in flux
                 self.ceph_manager.osd_admin_socket(osd, command=['dump_ops_in_flight'],
-                                     check_status=False, timeout=30)
+                                     check_status=False, timeout=30, stdout=DEVNULL)
```

Comment:
if we don't want the side effects of `do_dump_ops()`, probably we can just remove this method? i.e. partially revert 7a528763d126eaca90e40b02aad02ab9c3c92a32 ?